<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow{
	background-color: rgba(170, 255, 32, 10);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1061</width>
      <height>581</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="home">
     <property name="toolTipDuration">
      <number>-2</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#home{
	background-image: url(:/petshop.jpg);
}</string>
     </property>
     <widget class="QPushButton" name="Ajouterunecagebtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 24px;
background-color: grey;
border-radius: 15px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Ajouter une cage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouteranimalbtn">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>300</y>
        <width>221</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 24px;
background-color: grey;
border-radius: 15px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Ajouter un animal</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>931</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: grey;
font-size: 72px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>WELCOME TO THE ZOO</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Ajouterunecage">
     <property name="styleSheet">
      <string notr="true">QWidget#Ajouterunecage{
	background-image: url(:/petshop.jpg);
}</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter une Cage</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre maximal d'animaux :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description breve                :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nbr_max_cage">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_cage">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>390</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Ajouter une cage</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="desc_cage">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_cage">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
</string>
      </property>
      <property name="text">
       <string>reference </string>
      </property>
     </widget>
     <widget class="QTableView" name="afficher_cages">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>110</y>
        <width>451</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Editer </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherchecage">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche_btn">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>67</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
     <widget class="QPushButton" name="print">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>print</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Supprimercage">
     <property name="styleSheet">
      <string notr="true">QWidget#Supprimercage{
	background-image: url(:/petshop.jpg);
}</string>
     </property>
     <widget class="QComboBox" name="delid">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>120</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimercage">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>330</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="deldesc">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>240</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="delnbr_animaux">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>190</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>254</x>
        <y>190</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>nombre d'animaux par cage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Editer </string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifiercage">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>330</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ajouteranimal">
     <property name="styleSheet">
      <string notr="true">QWidget#ajouteranimal{
	background-image: url(:/petshop.jpg);
}</string>
     </property>
     <widget class="QLineEdit" name="refanimal">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Ajouter un animal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Modifier un Animal</string>
      </property>
     </widget>
     <widget class="QTableView" name="listedesanimaux">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>70</y>
        <width>451</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="race">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sexe">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>250</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femelle</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="id_cageA">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>310</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Race</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>310</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id du cage</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>420</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Ajouter un animal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_desc_3">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>40</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>race</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_desc_2">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>sexe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_desc">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>40</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>40</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Editeranimal">
     <property name="styleSheet">
      <string notr="true">QWidget#Editeranimal
{
	background-image: url(:/petshop.jpg);
}</string>
     </property>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Ajouter un animal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Modifier un Animal</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="delRace">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="delAnim">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="delnom">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>180</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="delsex">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>240</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femelle</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="delId_cageA">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>300</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ModifierAnimal">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>380</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DelAnimal">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>380</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 3px;
border: 2px solid black;</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>reference</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>130</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Race</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>240</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>300</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id cage</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Statanimaux">
     <property name="styleSheet">
      <string notr="true">QWidget#Statanimaux{
	background-image: url(:/petshop.jpg);
}</string>
     </property>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>591</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="homebtn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>540</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 12px;
background-color: grey;
border-radius: 15px;
border: 2px solid black;</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
